modular_citadel/code/game/gamemodes/gangs/gang_crayon.dm

```diff
@@ -0,0 +1,236 @@
+#define RANDOM_GRAFFITI "Random Graffiti"
```

Comment:
Modularization isn't necessary. You can demodularize this safely without issue.

Reply:
demodularized and deleted gang_crayon.

Reply:
unresolving this, I need to redo parts of this.

Reply:
Demodularized it, now regular people can use gang spraycans without causing a runtime, and they just fire off regularly. Gang members can use each others spraycans, and regular spraycan decals now can go over walls since you told me about the planes thing.